### PR TITLE
Update committer instructions.

### PR DESCRIPTION
**Pull request by phoglund**
Adding a notice about bandwidth restrictions and updated some old instructions about branching.

### CODE REVIEW

File: native-code/development/index.md [hunk 1]
```diff
@@ -59,7 +59,9 @@ For desktop development:
 
 See [Android][1] and [iOS][2] pages for separate instructions.
 
-
+NOTICE: if you get `Remote: Daily bandwidth rate limit exceeded for <ip>`, make
```
Comment (KaptenJansson) [inline native-code/development/index.md]: NOTICE was previously highlighted, did you accidentally remove that?

File: native-code/development/index.md [hunk 2]
```diff
@@ -148,30 +150,32 @@ To see available release branches, run:
...
 
 ~~~~~ bash
-fetch = +refs/branch-heads/*:refs/remotes/branch-heads/*
+git checkout -b my_branch refs/remotes/branch-heads/43
+gclient sync
 ~~~~~
 
-To create a local branch tracking a remote release branch (in this example,
-the 43 branch):
+NOTICE: depot_tools are not tracked with your checkout, so it's possible gclient
```
Comment (KaptenJansson) [inline native-code/development/index.md]: Highlight NOTICE?

**Review by KaptenJansson — APPROVED** [review]
Couple of nits otherwise LGTM.

File: native-code/development/index.md (hunk 2 above)
Comment (phoglund) [reply native-code/development/index.md]: Done

File: native-code/development/index.md (hunk 1 above)
Comment (phoglund) [reply native-code/development/index.md]: Done

**Pull request merged**